Solutions/RubrikSecurityCloud/Playbooks/RubrikRansomwareDiscoveryAndVMRecovery/azuredeploy.json

```diff
@@ -24,8 +25,8 @@
             "6. Repeat steps for other connections"
         ],
         "prerequisitesDeployTemplateFile": "Azure-Sentinel/Solutions/RubrikSecurityCloud/Playbooks/RubrikCustomConnector/Rubrik_custom_conn.json",
-        "lastUpdateTime": "2022-09-26T00:00:00.000Z",
-        "entities":  ["account","url"],
+        "lastUpdateTime":  "2022-01-20T00:00:00.000Z",
+        "entities":  ["account","url","api"],
```

Comment:
Hi @benmeadowcroft "API" isn't a valid entity, please check the following document --> https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#entity-mapping

Reply:
Thanks, the team should have resolved this in [a839e7f](https://github.com/Azure/Azure-Sentinel/pull/7352/commits/a839e7fbbfe41ed7fbd38f589e9397c1f38a5243)